src/components/AdjustVaultSummary.tsx

```diff
@@ -0,0 +1,146 @@
+  label: string;
+  left: string;
+  right: string;
+};
+
+const TableRowWithArrow = ({ label, left, right }: TableRowWithArrowProps) => {
+  return (
+    <tr className="text-[13px] leading-[21px]">
+      <td className="text-[#A3A5B9] text-left">{label}</td>
+      <td className="text-[#666980] font-black px-2">{left}</td>
+      <td className="text-[#666980] font-black px-2">&#8594;</td>
+      <td className="text-right text-[#666980] font-black">{right}</td>
+    </tr>
+  );
+};
+
+type Props = {
+  locked: Amount<'nat'>;
+  debt: Amount<'nat'>;
+  lockedValue: Amount<'nat'>;
+  managerId: string;
+};
+
+const AdjustVaultSummary = ({
+  locked,
+  debt,
+  lockedValue,
+  managerId,
+}: Props) => {
+  const canMakeOffer = true;
+  const adjustButtonLabel = 'Make Offer';
+
+  const { governedParams } = useVaultStore(state => ({
+    governedParams: state.vaultGovernedParams,
+  }));
+
+  const params = governedParams.get(managerId);
+
+  const collateralizationRatio = makeRatioFromAmounts(lockedValue, debt);
+
+  const { displayAmount, displayBrandPetname, displayPercent } =
+    useAtomValue(displayFunctionsAtom) ?? {};
+
+  return (
+    <div className="w-full pt-[28px] pb-3 bg-white rounded-[10px] shadow-[0_22px_34px_0_rgba(116,116,116,0.25)]">
+      <div className="px-8">
+        <h3 className="mb-4 font-serif text-[22px] font-extrabold leading-[35px]">
+          Vault Summary
+        </h3>
+        <h4 className="font-serif font-bold leading-[26px]">Confirm Details</h4>
+        <div className="w-full p-2">
+          <table className="w-full">
+            <tbody>
+              <TableRowWithArrow
+                label="Depositing"
+                left={`${displayAmount && displayAmount(locked, 2, 'locale')} ${
+                  displayBrandPetname && displayBrandPetname(locked.brand)
+                }`}
+                right="--"
+              />
+              <TableRowWithArrow
+                label="Borrowing"
+                left={`${displayAmount && displayAmount(debt, 2, 'locale')} ${
+                  displayBrandPetname && displayBrandPetname(debt.brand)
+                }`}
+                right="--"
+              />
+              <TableRowWithArrow
+                label="Collateralization Ratio"
+                left={`${
+                  displayPercent && displayPercent(collateralizationRatio, 0)
+                }%`}
+                right="--"
+              />
+            </tbody>
+          </table>
+        </div>
+        <div className="w-full h-[1px] bg-[#D8D8D8]"></div>
+        <div className="w-full p-2">
+          <table className="w-full">
+            <tbody>
+              <TableRow left="Min. Collateralization Ratio" right="--" />
+              <TableRow
+                left="Liquidation Ratio"
+                right={`${
+                  params &&
+                  displayPercent &&
+                  displayPercent(params.liquidationMargin, 0)
+                }%`}
```

Comment:
why doesn't `displayPercent` include the `%` char?

Reply:
When it's used to turn a ratio into a value for a number input it needs to not have a % char.